packages/client/src/ConfigTest.ts

```diff
@@ -20,23 +21,18 @@ const sideChainConfig = {
  */
 export const CONFIG_TEST: StreamrClientConfig = {
     network: {
-        trackers: [
-            {
-                id: '0xb9e7cEBF7b03AE26458E32a059488386b05798e8',
-                ws: `ws://${process.env.STREAMR_DOCKER_DEV_HOST || '127.0.0.1'}:30301`,
-                http: `http://${process.env.STREAMR_DOCKER_DEV_HOST || '127.0.0.1'}:30301`
-            }, {
-                id: '0x0540A3e144cdD81F402e7772C76a5808B71d2d30',
-                ws: `ws://${process.env.STREAMR_DOCKER_DEV_HOST || '127.0.0.1'}:30302`,
-                http: `http://${process.env.STREAMR_DOCKER_DEV_HOST || '127.0.0.1'}:30302`
-            }, {
-                id: '0xf2C195bE194a2C91e93Eacb1d6d55a00552a85E2',
-                ws: `ws://${process.env.STREAMR_DOCKER_DEV_HOST || '127.0.0.1'}:30303`,
-                http: `http://${process.env.STREAMR_DOCKER_DEV_HOST || '127.0.0.1'}:30303`
-            }
-        ],
-        webrtcDisallowPrivateAddresses: false,
-        iceServers: []
+        layer0: {
+            entryPoints: [{
+                id: 'entryPointBroker',
+                type: NodeType.NODEJS,
```

Comment:
We use the `CONFIG_TEST` value also in browser environments, and therefore it is not maybe possible to define this value as NODEJS. Was it so that the enum values are mainly about connectivity level: in browsers we have limited connectivity, and in Node we have better connectivity?

Reply:
The type value here points to the type of the entry point. Setting types for PeerDescriptors is no longer necessary after https://github.com/streamr-dev/network/pull/1719

Reply:
The value here refers to the type of the entry point. Setting the type is no longer necessary as of https://github.com/streamr-dev/network/pull/1719